llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -559,6 +562,7 @@ PassBuilder::buildO1FunctionSimplificationPipeline(OptimizationLevel Level,
   FPM.addPass(ADCEPass());
   FPM.addPass(
       SimplifyCFGPass(SimplifyCFGOptions().convertSwitchRangeToICmp(true)));
+  FPM.addPass(LifetimeMovePass());
```

Comment:
Surely this pass does not need to be scheduled 3 times in the `-O1` pipeline? Shouldn't it be sufficient to run it once per function?